doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_external_memory.rst

```diff
@@ -9,86 +7,86 @@ Firmware upgrade with external memory
 In the SUIT manifest, you can define a component representing the cache partition in the external memory.
 Within the ``suit-payload-fetch`` sequence, you can then store any fetched payload into any ``CACHE_POOL`` component.
 
-When the secure domain processes the ``suit-install`` sequence, issuing ``suit-directive-fetch`` on any non-integrated payload will instruct the secure domain firmware to search for a given URI in all cache partitions in the system.
-However, when such a cache partition is located in the external memory, the secure domain is unable to access the data.
+When the Secure Domain processes the ``suit-install`` sequence, issuing ``suit-directive-fetch`` on any non-integrated payload will instruct the Secure Domain firmware to search for a given URI in all cache partitions in the system.
+However, when such a cache partition is located in the external memory, the Secure Domain is unable to access the data.
 Before any ``suit-directive-fetch`` directive is issued that accesses a payload stored on the external memory, a companion image that implements an external memory device driver must be booted.
 
 The companion image consists of two main parts:
 
 * Device driver adequate for the external memory device
 
-* IPC service exposed towards the secure domain
+* IPC service exposed towards the Secure Domain
 
-When the companion image is booted and a directive that accesses the data on the external memory is issued, such as the ``suit-directive-fetch`` and ``suit-directive-copy`` directives, the secure domain firmware uses the IPC service provided by the companion image to access the contents of the external memory.
+When the companion image is booted and a directive that accesses the data on the external memory is issued, such as the ``suit-directive-fetch`` and ``suit-directive-copy`` directives, the Secure Domain firmware uses the IPC service provided by the companion image to access the contents of the external memory.
 Beyond the booting of the companion image, the update process does not differ from regular fetch model-based update.
 
 Enabling external flash support in SUIT DFU
 *******************************************
 
-The :file:`nrf/samples/suit/smp_transfer` sample contains a premade configuration enabling the external memory in SUIT DFU.
-To enable the external memory, you must apply the :file:`nrf/samples/suit/smp_transfer/overlay-extmem.conf` and :file:`nrf/samples/suit/smp_transfer/ext_flash_nrf54h20dk.overlay` overlays to the sample build or complete the following steps:
+The ``nrf/samples/suit/smp_transfer`` sample contains a premade configuration enabling the external memory in SUIT DFU.
+To enable the external memory, you must add the ``-DFILE_SUFFIX="extflash"`` argument to the build, or complete the following steps:
 
-1. Create a partition named ``companion_partition`` in the executable memory region that is accessible to the application domain by editing the devicetree (DTS) file:
```

Comment:
It seems that the information about the need of the `companion_partition` in the dts file was removed

Reply:
@e-rk Can you please confirm?
Based off this commit [here](https://github.com/nrfconnect/sdk-nrf/pull/15675/commits/224243dba76984b7e7067b4f49d76838bb68a05e)

Reply:
The companion partition configuration is now in the `flash_companion/README.md`. It is linked in the next point.